src/utils/string.ts

```diff
@@ -0,0 +1,3 @@
+export const uppercaseFirstChar = (str: string) => {
+  return str.charAt(0).toUpperCase() + str.substring(1);
+};
```

Comment:
Would something like this work here?

- https://lodash.com/docs/4.17.15#startCase

Reply:
I was also thinking if we can go without this function. I'd even prefer to use `text-transform: capitalize`, but if we follow the design precisely, we'll notice that the word "at" is lowercased there, so not to ruin that I decided to uppercase only the first character.

Reply:
Makes sense. Let's leave it as is then.

Reply:
Makes sense. Let's leave it as is then.